nixos/modules/services/networking/zerotierone.nix

```diff
@@ -69,13 +69,14 @@ in
     environment.systemPackages = [ cfg.package ];
 
     # Prevent systemd from potentially changing the MAC address
-    environment.etc."systemd/network/50-zerotier.link".text = ''
-      [Match]
-      OriginalName=zt*
-
-      [Link]
-      AutoNegotiation=false
-      MACAddressPolicy=none
-    '';
+    systemd.network.links."50-zerotier" = {
+      matchConfig = {
```

Comment:
How come the key is not "Match"? Is is not calling `lib.generators.toINI` under the hood?

Reply:
The home `systemd.network` attrset currently has `linkConfig`, `matchConfig`, `networkConfig` etc. Changing that is out of scope for this PR IMHO.

Reply:
Also systemd config files aren't strictly standard `toINI` compatible.